app/src/main/java/com/hover/stax/utils/DateUtils.kt

```diff
@@ -0,0 +1,198 @@
+package com.hover.stax.utils
+
+import android.content.Context
+import com.google.android.material.datepicker.MaterialDatePicker
+import com.hover.stax.R
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+object DateUtils {
+    const val MIN = 60 * 1000
+    const val DAY = 24 * 60 * MIN
+    @JvmStatic
+    fun now(): Long {
+        return Date().time
+    }
+
+
+
+    @JvmStatic
+    fun today(): Long {
+        return MaterialDatePicker.todayInUtcMilliseconds()
+    }
+
+    fun todayDate(): Date = Calendar.getInstance().time
+
+
+
+    fun timeAgo(context: Context, millis: Long): String {
+        return humanFriendlyTime(context, System.currentTimeMillis() - millis)
+    }
+
+    @JvmStatic
+    fun humanFriendlyDate(timestamp: Long): String {
+        if (timestamp == -1L) return ""
+        val now = Calendar.getInstance()
+        now.timeInMillis = now()
+        val date = Calendar.getInstance()
+        date.timeInMillis = timestamp
+        var str = monthNumToName(date[Calendar.MONTH])
+        str += " " + date[Calendar.DAY_OF_MONTH]
+        if (date[Calendar.YEAR] != now[Calendar.YEAR]) str += " " + date[Calendar.YEAR]
+        return str
+    }
+
+    fun humanFriendlyTime(context: Context, diffMillis: Long): String {
+        val seconds = (Math.abs(diffMillis) / 1000).toDouble()
+        val minutes = seconds / 60
+        val hours = minutes / 60
+        val days = hours / 24
+        val years = days / 365
+        val time: String
+        time = if (seconds < 45) {
```

Comment:
Can relative timestamp work for this?

Reply:
This isn't a new code. 
However, the customization was made so that we can extend more Date functions easily.